src/app/ApplicationTemplate.Wasm/ApplicationTemplate.Wasm.csproj

```diff
@@ -34,9 +37,9 @@
     <PackageReference Include="System.Reactive" Version="5.0.0" />
     <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.DeveloperTools" Version="6.1.0-build.205.g2e0e348372" />
     <PackageReference Include="Uno.UI.WebAssembly" Version="3.9.1" />
```

Comment:
Uno.UI.WebAssembly and the remotecontrol should be same version, I think 3.11.6 is used everywhere.